mmv1/products/kms/terraform.yaml

```diff
@@ -98,6 +98,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       decoder: templates/terraform/decoders/kms.go.erb
       encoder: templates/terraform/encoders/kms_crypto_key.go.erb
       update_encoder: templates/terraform/update_encoder/kms_crypto_key.go.erb
+  CryptoKeyVersion: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+
+      Destroying a cryptoKeyVersion will not delete the resource from the project.
+    id_format: "{{crypto_key}}"
+    import_format: ["{{crypto_key}}"]
+    skip_delete: true
```

Comment:
I think we probably don't want `skip_delete: true` here - the custom delete code should be sufficient.

Reply:
I remember adding it for fixing some failure while running tests, but I'll confirm now that I'm more familiar with the TF terms and process